modules/combined/examples/phase_field-mechanics/grain_texture.i

```diff
@@ -0,0 +1,165 @@
+# This simulation predicts GB migration of 8 grains and outputs grain texture information
+# Mesh adaptivity is not used so that the VectorPostprocessor's output will be uniform
+# Time step adaptivity is used
+# An AuxVariable is used to calculate the grain boundary locations
+# Postprocessors are used to record time step
+
+[Mesh]
+  # Mesh block.  Meshes can be read in or automatically generated
+  type = GeneratedMesh
+  dim = 2 # Problem dimension
+  nx = 100 # Number of elements in the x-direction
+  ny = 100 # Number of elements in the y-direction
```

Comment:
Good point - 100x100 is pretty large for a test. It might seem fast on your local workstation but it'll be dog slow in debug mode on a linux box with all the assertion tests. I'd try to cut it down as much as you can.